docs/section16/Rule 16-5.md

```diff
@@ -0,0 +1,34 @@
+# Elevators – Rule 16-5  
+**Schema Version:** 0.0.36        
+**Mandatory Rule:** True          
+**Rule ID:** 16-5  
+**Rule Description:** When included in the proposed design, the baseline elevator cab ventilation fan and lights shall operate continuously  
+**Rule Assertion:** B-RMD = Expected Value                                           
+**Appendix G Section:** G3.1  
+**Appendix G Section Reference:** Table G3.1-16  
+**Data Lookup:** None  
+**Evaluation Context:** Each elevator  
+
+**Applicability Checks:**  
+  1. The proposed building has an elevator.  
+
+**Function Call:**  
+N/A
+
+**Applicability Check 1:**
+- Rule is applicable if any building in the proposed RMD has an elevator: `if any(len(building.elevators) > 0 for building in P_RMD...buildings:`
+
+## Rule Logic:
+- For each elevator in the baseline RMD: `for elevator in B_RMD...elevators`
+  - Get the modeled schedule for the baseline elevator cab ventilation fan: `cab_ventilation_schedule_b = elevator.cab_ventilation_fan_multiplier_schedule`
+  - Get the modeled schedule for the baseline elevator cab lighting: `cab_lighting_schedule_b = elevator.cab_lighting_multiplier_schedule`
+  - Compare the schedules: `comparison_data = compare_schedules(cab_lighting_schedule_b, cab_ventilation_schedule_b):`  
```

Comment:
I see that this RDSs checks that the ventilation and cab lighting schedules were modeled the same but not that it checks that they were modeled as continuously operating. I think additional logic is needed to check that the schedules are equal to 1 for all hours as well.